puppet-client.html

```diff
@@ -162,16 +199,46 @@
                 value: '/'
             },
 
+            /**
+             * @private
```

Comment:
this property is used internally and wasan't meant to be exposed. We (together with @warpech ) just didn't find a way not to expose it